lib/screens/alerts/parser.ex

```diff
@@ -97,32 +97,17 @@ defmodule Screens.Alerts.Parser do
 
   defp parse_and_sort_active_periods(periods) do
     periods
-    |> Enum.map(&parse_active_period/1)
-    |> Enum.sort_by(fn {start, _} -> start end, fn dt1, dt2 ->
-      DateTime.compare(dt1, dt2) in [:lt, :eq]
+    |> Enum.map(fn %{"start" => start_str, "end" => end_str} ->
+      {parse_datetime(start_str), if(is_nil(end_str), do: nil, else: parse_datetime(end_str))}
     end)
+    |> Enum.sort_by(
+      fn {start, _} -> start end,
+      fn dt1, dt2 -> DateTime.compare(dt1, dt2) in [:lt, :eq] end
+    )
   end
 
-  defp parse_active_period(%{"start" => nil, "end" => end_str}) do
-    end_t = parse_time(end_str)
-    {nil, end_t}
-  end
-
-  defp parse_active_period(%{"start" => start_str, "end" => nil}) do
-    start_t = parse_time(start_str)
-    {start_t, nil}
-  end
-
-  defp parse_active_period(%{"start" => start_str, "end" => end_str}) do
-    start_t = parse_time(start_str)
-    end_t = parse_time(end_str)
-    {start_t, end_t}
-  end
-
-  defp parse_time(nil), do: nil
-
-  defp parse_time(s) do
-    {:ok, time, _} = DateTime.from_iso8601(s)
+  defp parse_datetime(iso_str) do
+    {:ok, time, _offset} = DateTime.from_iso8601(iso_str)
```

Comment:
In Elixir, is it just a stylistic preference to having "_" versus "_offset" here? 

I see in a lot of places we use plain ol' underscore, but I really do like the idea of having "_value" instead to be super explicit.

Reply:
I personally tend towards naming unused variables, and there is a [Credo check](https://github.com/rrrene/credo/blob/965a59c0e71067c7fc6fab26860ac6dc0ea2912f/lib/credo/check/consistency/unused_variable_names.ex#L1) we could enable if we wanted to enforce that, but I like having the option of keeping them anonymous. For example, `[_ | _]` is a common pattern used to match a non-empty list, and to me it wouldn't really make sense to name those variables, as the structure of the pattern is more important than the values themselves.